src/credentials/CHIPCert.cpp

```diff
@@ -618,6 +614,21 @@ void ChipCertificateData::Clear()
     memset(mTBSHash, 0, sizeof(mTBSHash));
 }
 
+bool ChipCertificateData::IsEqual(const ChipCertificateData & other) const
+{
+    return mSubjectDN.IsEqual(other.mSubjectDN) && mIssuerDN.IsEqual(other.mIssuerDN) &&
```

Comment:
I really wish we could have something that automatically did operator== on all the members so we don't have to update this if we add a member, but I don't see a way to do that in C++.